scripts/exec.js

```diff
@@ -42,10 +42,10 @@ function exec(cmd, displayName, cwd = process.cwd(), opts = {}) {
     );
 
     if (opts.stdout) {
-      child.stdout.pipe(opts.stdout);
+      child.stdout?.pipe(opts.stdout);
```

Comment:
I'm fine with this either way but I think this maintains the previous behavior (i.e., an error will be thrown when `child.stdout` is not defined)

```suggestion
      child.stdout!.pipe(opts.stdout);
```

Reply:
node 10: 
<img width="480" alt="image" src="https://user-images.githubusercontent.com/1223799/200525863-fb80c49b-b403-4884-809c-a4139a835cbe.png">

node 14:
<img width="428" alt="image" src="https://user-images.githubusercontent.com/1223799/200526405-d902c2a1-94f5-417c-a37a-4d33d122624a.png">


node 14 types correctly reflects the streamable behavior that is determined by `stdio` setting of spawn. 
This doesn't apply to `exec` though as you cannot set `stdio`. Also  because `exec` always spawns a shell  I/O redirection is always possible thus (stdout/stderr) will always be defined.  

**Summary:**
- this will never error on accessing non existing property.
- using `!` is antipattern  as I mentioned in other comment
- this whole "abstraction" is wrong and will be removed so I don't see issues with changes in this PR